pandas/core/indexes/base.py

```diff
@@ -1005,6 +1005,29 @@ def to_series(self, **kwargs):
                       index=self._shallow_copy(),
                       name=self.name)
 
+    def to_frame(self, index=True):
+        """
+        Create a DataFrame with the columns the levels of the Index
```

Comment:
I think this could be reworded.  Unless I'm missing something this should only return one column for the base `Index`, which should only have a single level (`MultiIndex` has it's own method/docstring).  So maybe something like "Create a DataFrame with the Index values as a column".

Reply:
Yeah...bad English copied and pasted.  Fixed.